src/tests/createHeadlessForm.test.js

```diff
@@ -2580,7 +2580,7 @@ describe('createHeadlessForm', () => {
       );
       fields = result.fields;
     });
-    describe('and file is of inccorrect format', () => {
+    describe('and file is of incorrect format', () => {
```

Comment:
@ollyd could you please review this? I was @ricardojoaomestre's buddy here, so we need an unbiased reviewer :)

Reply:
Hey @sandrina-p I didn't get a chance today - this will be the first thing I look at in the morning.